src/challenges.js

```diff
@@ -1,21 +1,37 @@
 /* ======================== CallBacks Practice ============================ */
 const each = (elements, cb) => {
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);
+  }
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
 };
 
 const map = (elements, cb) => {
+  var newString = [];
+  for (let i = 0; i < elements.length; i++) {
+    elements[i] += newString;
+  } return newString;
   // Produces a new array of values by mapping each value in list through a transformation function (iteratee).
   // Return the new array.
 };
 
 /* ======================== Closure Practice ============================ */
 const limitFunctionCallCount = (cb, n) => {
+  let i = 0;
+  return (... args) {
+    if (i < n) {
+      i++;
+      return cb(... args);
+    }
+    return null
+  }
   // Should return a function that invokes `cb`.
   // The returned function should only allow `cb` to be invoked `n` times.
 };
 
 const cacheFunction = cb => {
```

Comment:
This is incomplete.  It looks like you may have not committed everything for this one.